src/popup/locales/en.json

```diff
@@ -565,6 +565,7 @@
         "register-name": "Register .chain name",
         "name-placeholder": "Enter .chain name",
         "button": "Register name",
+        "button-price": "Register name for {price} AE",
```

Comment:
This isn't an unobvious to get the meaning of parameter here, for other places we've used default `{0}`, maybe let's stick with that as well